routes/api/bookin.js

```diff
@@ -5,10 +5,118 @@ const Hotel = require("../../models/Hotel");
 const Booking = require("../../models/booking");
 const Customer = require("../../models/customer");
 const City = require("../../models/city")
+const User = require("../../models/User")
 
 const checkAvailability = require('../../validation/checkAvailibility.js');
 const checkAvalibity = require("../../validation/checkAvailableHotels");
 
+// @route GET /api/booking/history
+// @desc History page
+// @access public
+router.get("/history", (req, res) => {
+    const userEmail = req.body.email;
+    var historyPack = [];
+
+    User.findOne({email: userEmail}).then((err, doc) => {
+        if(err) res.status(400).json(err)
+
+        // Uncomment below during production
+        // idToSearch = doc.customerID;
+
+        // Searching with a Customer ID of a User that has a booking
+        idToSearch = "5ca03969306f0952106d6eed";
+
+        Booking.find({customerID: idToSearch}).then((err, bookings) => {
```

Comment:
you can do Booking.find({customerID: idToSearch}).populate("hotelID").then(bookings =>{  }).catch(err)  to get the hotel info, so you don't need to call the database to look for hotel infos. info will contain in bookings.hotelID, or you can do a console.log(bookings) to check.